app/controllers/EmailController.scala

```diff
@@ -39,12 +39,16 @@ class EmailController @Inject()(authenticate: IdentifierAction,
   val log: LoggerLike = Logger(this.getClass)
 
   def showUnverified():Action[AnyContent] = authenticate async { implicit request =>
-    financialsApiConnector.isEmailUnverified.map(email => Ok(verifyEmailView(appConfig.emailFrontendUrl, email)))
+    dataStoreConnector.retrieveUnverifiedEmail.map(
+      emailUnverifiedRes => Ok(verifyEmailView(appConfig.emailFrontendUrl, 
```

Comment:
Please remove the empty whitespaces at the end of this line and generating scala style warning. Please move content on line 44 to line 43 (like below) as it does not make the line long.  Same for line 50 and 51

emailUnverifiedRes => Ok(verifyEmailView(appConfig.emailFrontendUrl, emailUnverifiedRes.unVerifiedEmail))